docs/contracts/node-operators-registry.md

```diff
@@ -14,126 +39,306 @@ Returns the rewards distribution proportional to the effective stake for each no
 ```sol
 function getRewardsDistribution(uint256 _totalRewardShares) returns (
   address[] recipients,
-  uint256[] shares
+  uint256[] shares,
+  bool[] penalized
 )
 ```
 
-#### Parameters:
+#### Parameters
```

Comment:
Also, I eliminated repeating headers like this in lido.md. I think it looks clearer, but it's discussable.